sdk/src/main/scala/io/horizen/account/AccountSidechainApp.scala

```diff
@@ -55,7 +55,8 @@ class AccountSidechainApp @Inject()
    @Named("ApplicationStopper") applicationStopper: SidechainAppStopper,
    @Named("ForkConfiguration") forkConfigurator: ForkConfigurator,
    @Named("ChainInfo") chainInfo: ChainInfo,
-   @Named("ConsensusSecondsInSlot") secondsInSlot: Int
+   @Named("ConsensusSecondsInSlot") secondsInSlot: Int,
+   @Named("EonVersion") eonVersion: String
```

Comment:
At this point that we generalized the name, can we use it also inside the UTXO model?